.envrc

```diff
@@ -0,0 +1,28 @@
+layout node
+
+# You can define your system-specific logic (like Git settings or GH tokens) in .envrc.local
+# If that logic is usable by other people and might improve development environment, consider
+# contributing it to this file ♥
+
+watch_file .envrc.local
+[ -f .envrc.local ] && {
+    source .envrc.local    
+}
```

Comment:
i never quite see the upside of this short form. also we know we're using bash so we can prefer `[[`

```suggestion
if [[ -f .envrc.local ]] ; then
    source .envrc.local    
fi
```